Mathlib/SetTheory/ZFC/Rank.lean

```diff
@@ -122,6 +125,8 @@ theorem rank_eq_wfRank : lift.{u + 1, u} (rank x) = IsWellFounded.rank (α := PS
 
 end PSet
 
+/-! ### ZFSet rank -/
+
 namespace ZFSet
 
 variable {x y : ZFSet.{u}}
```

Comment:
```suggestion
```

I think this line can be removed now (and add implicit arguments for `rank_lt_of_mem` and so on), since explicit arguments are used in most cases.

It's the same for `PSet`, although Github doesn't allow me to comment on that line.

Reply:
Actually, enough theorems still use some implicit variables that I think having the `variable` block is still useful.

Reply:
That makes sense.